frontend/src/components/navigation/NavHeader.tsx

```diff
@@ -188,8 +189,9 @@ export default function NavHeader({
                 </div>
               ) : (
                 <Link
-                  to="/projects/secret-management/$projectId/secrets/$envSlug"
```

Comment:
Ignore comment placement

All the assume privilege redirect are broken, this is project user, identity and group detail page

<img width="1746" height="640" alt="Image" src="https://github.com/user-attachments/assets/abe1ece2-fb16-4097-a7f8-6b2ced369e07" />